### ISSUE TITLE
RayCast debug shape culled by Rooms and Portals (when Visible Collision Shapes is enabled)

### ISSUE THREAD

**Issue by Sirus97** [godotengine/godot#65673]
### Godot version

v3.5.stable.official [991bb6ac7]

### System information

Windows 10, GLES3

### Issue description

Room conversion culls RayCast's debug shape.

Possible solutions:
1. Access `CullInstance` from the inspector
2. Default to `PORTAL_MODE_GLOBAL`

![image](https://user-images.githubusercontent.com/86083633/189553911-8e703a50-38e6-4163-bfbf-b81bf7f51383.png)

### Steps to reproduce

1. Enable Visible Collision Shapes.
2. Add RayCast in 3D environment and camera to render it.
3. Set up Rooms and Portals in the same environment.
4. Run the scene.

![image](https://user-images.githubusercontent.com/86083633/189554048-7c902043-57b4-4204-9fd7-928c4d3f5df5.png)

```
extends Spatial

func _ready():
	pass
	$RoomManager.rooms_convert()

```

### Minimal reproduction project

[RayCast Bug.zip](https://github.com/godotengine/godot/files/9543772/RayCast.Bug.zip)

**Comment by Calinou** [comment]
cc @lawnjelly

**Comment by lawnjelly** [comment]
I can switch this to work, but in general, a lot of debugging stuff will not show up when portals are active. I tried to mention this in the docs, I found this for `RoomManager::active`, and I'm sure I mentioned it elsewhere:

> It is important to note that when portal culling is active, it is responsible for all the 3d culling. Some editor functionality may be more difficult to use, so switching the active flag is intended to be used to make sure your Room / Portal layout works within the editor.

Also see here:
https://docs.godotengine.org/en/stable/tutorials/3d/portals/first_steps_with_rooms_and_portals.html#room-conversion

> If you convert the level while the editor is running, the portal culling system will take over from the normal Godot frustum culling, potentially interfering with editor features.

There is likely quite a bit of debugging stuff that does not show up, during debugging you are recommended to not activate portals. Once you are finished debugging, then reactivate. It is not really _a bug_, more a design consequence. I can individually switch these elements to show but it adds more code for not great reasons.

Of course there might be a situation where you want to use the debugging objects to debug your portal setup, in which case I do sympathize. :grin: 

Incidentally the reason for this is that editor debuggers (and debuggers you can switch to show during runtime) are not regular `VisualInstances`. They often create RIDs through the `VisualServer` directly, and need to have a portal mode set for the system to understand them and decide whether or not to show them.

**Comment by akien-mga** [comment]
Fixed by #65684 and #65686.
